dbtwiz/commands/build.py

```diff
@@ -0,0 +1,116 @@
+from datetime import date, timedelta
+import json
+import os
+from pathlib import Path
+
+from google.cloud import storage
+
+from dbtwiz.auth import ensure_auth
+from dbtwiz.config import project_config, project_dbtwiz_path
+from dbtwiz.dbt import dbt_invoke
+from dbtwiz.logging import info, debug, error, fatal
+from dbtwiz.manifest import Manifest
+
+
+VALID_TARGETS = ["dev", "build", "prod-ci", "prod"]
+
+LAST_SELECT_FILE = project_dbtwiz_path("last_select.json")
+
+
+def build(target: str,
```

Comment:
Function description? (here and for all the functions)

Reply:
Especially since there are many parameters that could do with some describing.

Reply:
Create an issue for adding descriptions on top and for all functions missing it.

Reply:
I've added an issue for this: https://github.com/amedia/dbtwiz/issues/28

Though all new functions in this PR should be given a description as a part of this PR, while descriptions for all existing functions that are only moved can be added later on.